static/js/narrow.js

```diff
@@ -1076,4 +1080,6 @@ export function deactivate(coming_from_recent_topics = false) {
         unnarrow_times.initial_free_time = new Date();
         report_unnarrow_time();
     });
+
+    unread_ui.reset_mark_as_read_turned_off_banner();
```

Comment:
This is an odd place to add code; I'm not sure whether `handle_post_narrow_deactivate_processes` is where it belongs, or whether this function calls `reset_ui_state`, but we don't have a lot of direct UI changes in this function, so it seems likely misplaced?

Reply:
Oh this function already calls `reset_ui_state` 😅 

Thanks for calling that out!